ui/app/styles/utils/_size_variables.scss

```diff
@@ -26,101 +25,42 @@ $size-normal: $size-6;
 $size-medium: $size-5;
 $size-large: $size-4;
 
-/* Font weight */
-$weight-light: 300 !default;
-$weight-normal: 400 !default;
-$weight-medium: 500 !default;
-$weight-semibold: 600 !default;
-$weight-bold: 700 !default;
-$font-weight-normal: 400;
-$font-weight-semibold: 600;
-$font-weight-bold: 700;
-
 /* Spacing */
-$block-spacing: 1.5rem !default;
-
 $spacing-xxs: 4px;
 $spacing-xs: 8px;
 $spacing-s: 12px;
 $spacing-m: 16px;
 $spacing-l: 24px;
 $spacing-xl: 36px;
 $spacing-xxl: 48px;
+$spacing-xxxl: 64px;
 
-/* Misc. */
-$column-gap: 0.75rem !default;
-// The container horizontal gap, which acts as the offset for breakpoints
-$gap: 32px !default;
-// 960, 1152, and 1344 have been chosen because they are divisible by both 12 and 16
+/* Font weight */
+$font-weight-normal: 400;
+$font-weight-semibold: 600;
+$font-weight-bold: 700;
+
+/* Border radius */
+$radius: 2px;
+$radius-large: 4px;
 
 /* Responsiveness */
+// 960, 1152, and 1344 have been chosen because they are divisible by both 12 and 16
 $mobile: 769px;
-$tablet: 769px !default;
 // 960px container + 4rem
-$desktop: 960px + (2 * $gap) !default;
-// 1152px container + 4rem
-$widescreen: 1152px + (2 * $gap) !default;
-$widescreen-enabled: true !default;
+$desktop: 960px + (2 * 32px);
 // 1344px container + 4rem
-$fullhd: 1344px + (2 * $gap) !default;
-$fullhd-enabled: true !default;
-$breakpoints: (
-  'mobile': (
-    'until': $tablet,
-  ),
-  'tablet': (
-    'from': $tablet,
-  ),
-  'tablet-only': (
-    'from': $tablet,
-    'until': $desktop,
-  ),
-  'touch': (
-    'from': $desktop,
-  ),
-  'desktop': (
-    'from': $desktop,
-  ),
-  'desktop-only': (
-    'from': $desktop,
-    'until': $widescreen,
-  ),
-  'until-widescreen': (
-    'until': $widescreen,
-  ),
-  'widescreen': (
-    'from': $widescreen,
-  ),
-  'widescreen-only': (
-    'from': $widescreen,
-    'until': $fullhd,
-  ),
-  'until-fullhd': (
-    'until': $fullhd,
-  ),
-  'fullhd': (
-    'from': $fullhd,
-  ),
-) !default;
-
-/* Miscellaneous sizing*/
-// ARG TODO clean up
-$radius-small: 2px !default;
```

Comment:
I know what this looks like, it's wrong. We imported bulma_variables after this file so we actually overrode these. 🤯